moveit_core/CMakeLists.txt

```diff
@@ -1,68 +1,66 @@
 cmake_minimum_required(VERSION 3.10.2)
-project(moveit_core)
+project(moveit_core VERSION 2.0.0 LANGUAGES CXX)
 
-add_compile_options(-std=c++14)
```

Comment:
Are you sure the code below adds the flag `-std=c++14`? It may only check whether the C++ compiler is capable of compiling C++14. It could add a flag like `-std=gnu++14`, which may not be what you want. See https://cmake.org/cmake/help/latest/prop_tgt/CXX_STANDARD.html.

Reply:
True, MoveIt is supposed to use the strict standard implementation, so you need to add 
```
set(CMAKE_CXX_EXTENSIONS OFF)
```

Reply:
Actually, I'm intrigued to just revert this change and just update it with the upcoming sync, keeping the original commit.